### PR TITLE
op-guide, media: update monitoring docs

### PR DESCRIPTION
**Pull request by lilin90**
- Updated TiDB monitoring documents based on the user acceptance testing
  - Updated wording
  - Replaced outdated content with the new content
- Reorganized TiDB monitoring documents based on [Morgan's comment](https://github.com/pingcap/docs/pull/944#issuecomment-471652192)

**Two points:**

1. I checked with the DBA and got that in TiDB v3.0-beta and v2.1, Pushgateway is not used anymore (not supported) in  PD and TiDB, while it's optional in TiKV. 

    In this PR, I add a statement and keep the Pushgateway section. I didn't use Pushgateway in my testing. Removing it might make the document/process much clearer and simpler. Do we need to remove this Pushgateway section in v3.0 docs?

2. I learned that the [JSON file link](https://grafana.com/tidb) (in the current document) is not up-to-date. Could we use the [files in the tidb-ansible repo](https://github.com/pingcap/tidb-ansible/tree/master/scripts)? @liubo0127 

Related PR: #944 

@liubo0127 @morgo @c4pt0r @dcalvin PTAL

### CODE REVIEW

File: op-guide/monitor.md [hunk 1]
```diff
@@ -6,153 +6,266 @@ category: operations
...
 
-- the duration that a command finishes and returns result
+You can use the metrics interface to monitor the state and performance of the entire TiDB cluster.
 
-### TiKV server
+- If you use Ansible to deploy the TiDB cluster, the monitoring system (Prometheus and Grafana) is deployed at the same time.
+- If you use other deployment ways, [deploy Prometheus and Grafana](#deploy-prometheus-and-grafana) before using this interface.
 
-- Garbage Collection (GC) monitoring
+In TiDB 3.0, the Prometheus Pushgateway is **not used** by default, but you can also use it if you like. In earlier TiDB versions, the Prometheus Pushgateway is **used** by default, but you can also choose to not use it.
```
Comment (dcalvin) [inline op-guide/monitor.md]: Add it as a note here with a brief introduction of ** Pushgateway**

File: op-guide/monitor.md [hunk 2]
```diff
@@ -6,153 +6,266 @@ category: operations
...
 
-The PD API address: `http://${host}:${port}/pd/api/v1/${api_name}`
-
-The default port: `2379`
+- The PD API address: `http://${host}:${port}/pd/api/v1/${api_name}`
+- The default port: `2379`
+- The details about API names: see [PD API doc](https://cdn.rawgit.com/pingcap/docs/master/op-guide/pd-api-v1.html)
 
-See [PD API doc](https://cdn.rawgit.com/pingcap/docs/master/op-guide/pd-api-v1.html) for detailed information about various API names.
+You can use this interface to get the state of all the TiKV servers and the information about load balancing. It is the most important and frequently-used interface to get the state information of all the TiKV nodes.
```
[the remark this answers is not in the document]
Comment (lilin90) [reply op-guide/monitor.md]: @dcalvin I also think this sentence is a little bit weird here. Let's remove it.

**Comment by lilin90** [comment]
@dcalvin Thanks! Updated. PTAL again~

**Comment by liubo0127** [comment]
- Both 2.1 and master (3.0) support the pull mode, so you can remove pushgateway
- Can import grafana dashboard using json in tidb-ansible/scripts

File: op-guide/monitor.md [hunk 3]
```diff
@@ -179,69 +290,102 @@ See the following links for your reference:
     job = "tikv"
     ```
 
-### Configure PushServer
+#### Configure PushServer
 
 Generally, it does not need to be configured. You can use the default port: 9091.
```
Comment (morgo) [inline op-guide/monitor.md]:
```suggestion
Generally, the PushServer does not need to be configured, since the default port of `9091` can be used.
```

**Comment by morgo** [comment]
There was one pre-existing sentence I improved for clarity.  Otherwise LGTM

**Comment by lilin90** [comment]
@liubo0127 Then I'll remove the Pushgateway section and update the JSON file link.

**Comment by lilin90** [comment]
@liubo0127 @morgo Thanks! Updated. PTAL again~

File: op-guide/monitor.md [hunk 4]
```diff
@@ -6,242 +6,314 @@ category: operations
...
-    interval = "15s"
-    # Prometheus Pushgateway address, leaves it empty will disable prometheus.
-    address = "host:port"
+    # Starts the node_exporter service.
+    $ ./node_exporter --web.listen-address=":9100" \
+        --log.level="info" &
     ```
 
-+ TiKV: update the toml configuration file with the Pushgateway address and the the push frequency. Set the job field as "tikv".
+3. Start Prometheus in Node1.
```
Comment (dcalvin) [inline op-guide/monitor.md]:
```suggestion
3. Start Prometheus for Node1.
```

File: op-guide/monitor.md [hunk 5]
```diff
@@ -6,242 +6,314 @@ category: operations
...
+            --config.file="./prometheus.yml" \
+            --web.listen-address=":9090" \
+            --web.external-url="http://192.168.199.113:9090/" \
+            --web.enable-admin-api \
+            --log.level="info" \
+            --storage.tsdb.path="./data.metrics" \
+            --storage.tsdb.retention="15d" &
+        ```
+
+4. Start Grafana in Node1.
```
Comment (dcalvin) [inline op-guide/monitor.md]:
```suggestion
4. Start Grafana on Node1.
```

File: op-guide/monitor.md [hunk 6]
```diff
@@ -6,242 +6,314 @@ category: operations
...
+    - Default address: [http://localhost:3000](http://localhost:3000)
+    - Default account: admin
+    - Default password: admin
 
-  # Override the global default and scrape targets from this job every 5 seconds.
-  scrape_interval: 5s
+2. Click the Grafana logo to open the sidebar menu.
 
-  honor_labels: true
+3. Click **Data Source** in the sidebar.
```
Comment (dcalvin) [inline op-guide/monitor.md]:
```suggestion
3. In the sidebar menu, click **Data Source**.
```

File: op-guide/monitor.md [hunk 7]
```diff
@@ -6,242 +6,314 @@ category: operations
...
+#### Step 2: Import a Grafana dashboard
 
-    - The default account name: admin
+To import a Grafana dashboard for the PD server, the TiKV server, and the TiDB server, take the following steps respectively:
 
-    - The password for the default account: admin
+1. Click the Grafana logo to open the sidebar menu.
 
-2. Click the Grafana logo to open the sidebar menu.
+2. On the sidebar menu, click **Dashboards** -> **Import** to open the **Import Dashboard** window.
```
Comment (dcalvin) [inline op-guide/monitor.md]:
```suggestion
2. In the sidebar menu, click **Dashboards** -> **Import** to open the **Import Dashboard** window.
```

File: op-guide/monitor.md [hunk 8]
```diff
@@ -6,242 +6,314 @@ category: operations
...
+5. Select a Prometheus data source.
 
-    - For Type, select Prometheus.
+6. Click **Import**. A Prometheus dashboard is imported.
 
-    - For Url, specify the Prometheus address.
+### View component metrics
 
-    - Specify other fields as needed.
+Click **New dashboard** in the top menu and choose the dashboard you want to view.
```
Comment (dcalvin) [inline op-guide/monitor.md]:
```suggestion
In the top menu, click **New dashboard** and choose the dashboard you want to view.
```

**Pull request merged**